product_internal_reference_code/models/res_company.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    def create(self, vals):
```

Comment:
```suggestion
    @api.model
    def create(self, vals):
```

Missed to catch this but it should be `@api.model`.